lib/LaTeXML/Package/latexml.sty.ltxml

```diff
@@ -44,6 +44,9 @@ DeclareOption('noprofiling', sub { AssignValue('PROFILING' => 0, 'global'); });
 DeclareOption('mathparserspeculate',   sub { AssignValue('MATHPARSER_SPECULATE' => 1, 'global'); });
 DeclareOption('nomathparserspeculate', sub { AssignValue('MATHPARSER_SPECULATE' => 0, 'global'); });
 
+DeclareOption('xpointer_locators', sub { AssignValue('LOCATOR_STYLE' => 'xpointer', 'global'); });
+DeclareOption('locator_narrative', sub { AssignValue('LOCATOR_STYLE' => 'narrative', 'global'); });
```

Comment:
And one last bit here is making the names consistent, e.g. ```narrative_locators``` instead of ```locator_narrative```. Though Bruce may have better suggestions than me.